skyVegasTests/depositFLowGameClient.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { test, type Page } from "@playwright/test";
+import { launchGame, validateConsoleMessages, login } from "./utilities";
+import * as fs from "fs";
+import { config } from "../config/config";
+import { startEventListener } from "./utilities/starteventListenerHelper";
+
+test.describe.configure({ mode: "serial" });
```

Comment:
Since its a single test, dont think you need the serial mode.

Reply:
removed now